src/content/operations/usage/_index.en.md

```diff
@@ -687,6 +687,11 @@ Kubernetes supports this by introducing stable Pod IDs composed of `<pod-name>.<
 which Submariner extends to `<pod-name>.<cluster-id>.<svc-name>.<ns>.svc.clusterset.local` across the cluster set. The Headless Service in
 this case offers one single Service for all the underlying Pods.
 
+{{% notice note %}}
+Since we need to use `<cluster-id>` in DNS query for individual pods, cluster ID must be a valid
+[DNS-1123 Label](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)
```

Comment:
```suggestion
Since we need to use `<cluster-id>` in DNS queries for individual pods, the cluster ID must be a valid
[DNS-1123 Label](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names).
```